Kickstarter-iOS/Library/CookieRefTagFunctions.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import KsApi
+
+private let cookieSeparator = "?"
+private let escapedCookieSeparator = "%3F"
+
+// Extracts the ref tag stored in cookies for a particular project. Returns `nil` if no such cookie has
+// been previously set.
+public func cookieRefTagFor(project: Project) -> RefTag? {
```

Comment:
Moved these functions to a shared place now that we need to access `cookieRefTagFor` in two places. Now that they're shared we may want to add tests for them specifically but I figured we could tackle them separately as this doesn't reduce our test coverage currently.